hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestOzoneConfigurationFields.java

```diff
@@ -133,6 +133,7 @@ private void addPropertiesNotInXml() {
         OMConfigKeys.OZONE_RANGER_HTTPS_ADDRESS_KEY,
         OMConfigKeys.OZONE_OM_RANGER_HTTPS_ADMIN_API_USER,
         OMConfigKeys.OZONE_OM_RANGER_HTTPS_ADMIN_API_PASSWD,
+        OMConfigKeys.OZONE_OM_ENABLE_OFS_SHARED_TMP_DIR,
```

Comment:
Should we add this new config key in `ozone-default.xml` instead of skipping the check here?

Reply:
Adding `ozone.om.enable.ofs.shared.tmp.dir` to `ozone-default.xml`.  Currently default value set to false.  Should it remain, or enable it by default?